includes/validation/class-amp-invalid-url-post-type.php

```diff
@@ -894,6 +894,34 @@ public static function print_admin_notice() {
 				esc_html__( 'Dismiss this notice.', 'amp' )
 			);
 		}
+
+		if ( 'post' !== get_current_screen()->base ) {
+			// Display admin notice according to the AMP mode.
+			$theme_support     = AMP_Options_Manager::get_option( 'theme_support' );
+			$auto_sanitization = AMP_Options_Manager::get_option( 'force_sanitization' );
+
+			if ( 'native' === $theme_support ) {
+				$message = __( 'The site is using native AMP mode, the validation errors found are already automatically handled.', 'amp' );
+			} elseif ( 'paired' === $theme_support && $auto_sanitization ) {
```

Comment:
Likewise, it might be more accurate to use `AMP_Theme_Support::is_paired_available()` to detect Paired Mode. That will detect when a theme registers Paired Mode, as well as when a user selects it via 'AMP Settings' > 'Template Mode.'

Reply:
Actually `AMP_Theme_Support::is_paired_available()` won't work because it is specifically intended to be called on the frontend in the context of a given `WP_Query`. I think that checking for `AMP_Options_Manager::get_option( 'theme_support' )` is correct.

Reply:
On second thought, per the comment above and whether theme support was added via `add_theme_support()`... 

```php
if ( amp_is_canonical() ) {
	$template_mode = 'native';
} elseif ( current_theme_supports( 'amp' ) ) {
	$template_mode = 'paired';
} else {
	$template_mode = 'classic';
}
```